epub34/authoring/index.html

```diff
@@ -498,14 +513,31 @@ <h3>Terminology</h3>
 					</dd>
 
 					<dt>
-						<dfn class="export" data-lt="xhtml content document|xhtml content documents">HTML content
-							document</dfn>
+						<dfn class="export">HTML content document</dfn>
+					</dt>
+					<dt>
+						<small>(formerly <dfn class="export">XHTML content document</dfn>)</small>
 					</dt>
 					<dd>
 						<p>An [=EPUB content document=] that conforms to the profile of [[html]] defined in <a
 								href="#sec-xhtml"></a>.</p>
 						<p>HTML content documents can be authored in either the <a data-cite="html#syntax">HTML
```

Comment:
```suggestion
						<p>HTML content documents can be expressed in either the <a data-cite="html#syntax">HTML
```

The word "authored" sounded funny to my ears here. Feel free to ignore me...

Reply:
The act is authoring but what you write is expressing, I suppose. Doesn't really make a difference to me.